### ISSUE TITLE
autopep8 not showing up in toolbar

### ISSUE THREAD

**Issue by Dylan-C-Neal** [ipython-contrib/jupyter_contrib_nbextensions#1522]
Maybe I'm not doing something correctly, but when enabling autopep8 it won't show up in the tool bar. The hot keys don't work either. I confirmed that I have autopep8 installed via conda, so I'm not sure what's wrong.

**Comment by MySlientWind** [comment]
I have the same problem...  Reinstall don't solve it.

**Comment by MySlientWind** [comment]
> Maybe I'm not doing something correctly, but when enabling autopep8 it won't show up in the tool bar. The hot keys don't work either. I confirmed that I have autopep8 installed via conda, so I'm not sure what's wrong.

Did you upgrade before this problem appear？

**Comment by dewith** [comment]
In my case it's happening with all **_code prettify extensions_** (code prettify, autopep8, isort). Same issue is described here #1527. Any possible solution?

**Comment by MySlientWind** [comment]
> In my case it's happening with all **_code prettify extensions_** (code prettify, autopep8, isort). Same issue is described here #1527.
> ![image](https://user-images.githubusercontent.com/65407698/93006847-f2d79b00-f526-11ea-8c7d-7c307bc24cee.png)
> ![image](https://user-images.githubusercontent.com/65407698/93006853-05ea6b00-f527-11ea-81be-462a96160b92.png)
> Any possible solution?

Me too.
And I fixed it by installing the last version of anaconda, and never 'conda upgrade --all'. Maybe **code prettify extensions** is incompatible with some new packages.

**Comment by dewith** [comment]
Thanks for the answer. But I reinstalled Anaconda just yesterday due to other issues, so I assume I have the latest version. Could you tell me exactly what version does not generate this problem?

**Comment by dewith** [comment]
> > In my case it's happening with all **_code prettify extensions_** (code prettify, autopep8, isort). Same issue is described here #1527.
> > ![image](https://user-images.githubusercontent.com/65407698/93006847-f2d79b00-f526-11ea-8c7d-7c307bc24cee.png)
> > ![image](https://user-images.githubusercontent.com/65407698/93006853-05ea6b00-f527-11ea-81be-462a96160b92.png)
> > Any possible solution?
> 
> Me too.
> And I fixed it by installing the last version of anaconda, and never 'conda upgrade --all'. Maybe **code prettify extensions** is incompatible with some new packages.

Thanks for the answer. But I reinstalled Anaconda just yesterday due to other issues, so I assume I have the latest version. Could you tell me exactly what version does not generate this problem?

**Comment by MySlientWind** [comment]
> > > In my case it's happening with all **_code prettify extensions_** (code prettify, autopep8, isort). Same issue is described here #1527.
> > > ![image](https://user-images.githubusercontent.com/65407698/93006847-f2d79b00-f526-11ea-8c7d-7c307bc24cee.png)
> > > ![image](https://user-images.githubusercontent.com/65407698/93006853-05ea6b00-f527-11ea-81be-462a96160b92.png)
> > > Any possible solution?
> > 
> > 
> > Me too.
> > And I fixed it by installing the last version of anaconda, and never 'conda upgrade --all'. Maybe **code prettify extensions** is incompatible with some new packages.
> 
> Thanks for the answer. But I reinstalled Anaconda just yesterday due to other issues, so I assume I have the latest version. Could you tell me exactly what version does not generate this problem?

2020.07, did you update some packages after  reinstalling?

**Comment by dewith** [comment]
> > > > In my case it's happening with all **_code prettify extensions_** (code prettify, autopep8, isort). Same issue is described here #1527.
> > > > ![image](https://user-images.githubusercontent.com/65407698/93006847-f2d79b00-f526-11ea-8c7d-7c307bc24cee.png)
> > > > ![image](https://user-images.githubusercontent.com/65407698/93006853-05ea6b00-f527-11ea-81be-462a96160b92.png)
> > > > Any possible solution?
> > > 
> > > 
> > > Me too.
> > > And I fixed it by installing the last version of anaconda, and never 'conda upgrade --all'. Maybe **code prettify extensions** is incompatible with some new packages.
> > 
> > 
> > Thanks for the answer. But I reinstalled Anaconda just yesterday due to other issues, so I assume I have the latest version. Could you tell me exactly what version does not generate this problem?
> 
> 2020.07, did you update some packages after reinstalling?

Yes, I downloaded Anaconda (2020-07), and installed every package I need with the latest version. I think it's a compatibility problem here.

**Comment by caoyang1024** [comment]
This is the reason I like Microsoft product. spent whole afternoon just try to prettify some code in Jupiter, crazy...
